pyproject.toml

```diff
@@ -33,6 +33,8 @@ dependencies = [
     "uvicorn>=0.31.1; sys_platform != 'emscripten'",
     "jsonschema>=4.20.0",
     "pywin32>=310; sys_platform == 'win32'",
+    "typing-extensions>=4.9.0",
+    "typing-inspection>=0.4.1",
```

Comment:
`typing-extensions` is already used in the project, but not declared as an explicit dependency. 4.9.0 is the oldest version with (proper) support for `typing_extensions.deprecated`.

`typing-inspection` is already a Pydantic dependency, so nothing new added here.